### PR TITLE
[browser-sync] Added cwd, callbacks, listen and script types to BrowserSync options

### PR DESCRIPTION
**Pull request by AndreasBomholtz**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <[#option-callbacks](https://browsersync.io/docs/options#option-callbacks)>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by joeskeen** [comment]
It's really hard to see what actually changed through all the formatting changes. Could you resubmit a PR with minimal changes or at least tell us where the relevant changes are?

**Comment by AndreasBomholtz** [comment]
Yes I can remove the "lining".  Should I open a new PR with just the lining?

**Comment by AndreasBomholtz** [comment]
@joeskeen I have now removed my linting commit, and it is now clear what changed.

File: types/browser-sync/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-// Type definitions for browser-sync 2.26
+// Type definitions for browser-sync 2.27
 // Project: http://www.browsersync.io/
 // Definitions by: Asana <https://asana.com>,
 //                 Joe Skeen <https://github.com/joeskeen>
 //                 Thomas "Thasmo" Deinhamer <https://thasmo.com/>
 //                 Kiyotoshi Ichikawa <https://github.com/aznnomness>
 //                 Yuma Hashimoto <https://github.com/yuma84>
+//                 Andreas Bomholtz <https://github.com/seluxit>
```
Comment (jakebailey) [inline types/browser-sync/index.d.ts:8]: Note that this adds you as an owner of the package, will ping you for reviews, etc. If you're just looking for contribution credit, we use the git history to manage that, so you don't need to provide this if you don't want to become an owner.

**Pull request merged**